stage0/run.go

```diff
@@ -262,12 +263,12 @@ func Prepare(cfg PrepareConfig, dir string, uuid *types.UUID) error {
 
 	log.Printf("Writing pod manifest")
 	fn := common.PodManifestPath(dir)
-	if err := ioutil.WriteFile(fn, pmb, 0700); err != nil {
+	if err := ioutil.WriteFile(fn, pmb, 0660); err != nil {
```

Comment:
Can you split the perm out into a documented const to make its significance more obvious?